webserver/templates/user/import.html

```diff
@@ -37,6 +37,22 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
         </div>
      {% else %}
      <p>
+        <strong>Notes about the Last.fm import process:</strong>
+     </p>
+     <p>
+        The Last.fm importer manually steps through your listen history and imports the listens
```

Comment:
Here would be a place to additionally add 

 - Don't go away from this tab when it's running
 - It's better to not be listening/scrobbling at the same time

if wanted